mods/cnc/chrome/lobby-options.yaml

```diff
@@ -53,6 +53,12 @@ Background@LOBBY_OPTIONS_BIN:
 					Height: 20
 					Font: Regular
 					Text: Debug Menu
+				Checkbox@BUILDRADIUS_CHECKBOX:
+					Y: 70
+					Width: 150
+					Height: 20
+					Font: Regular
+					Text: Build Radius
```

Comment:
"Build Radius" isn't a very obvious name.  Can we change this to something like "Limit ConYard Area"?